index.html

```diff
@@ -19,7 +19,7 @@
                 &lt;script src='stream.min.js'&gt;&lt;/script&gt;
             </pre>
 
-            <a class='download' href="stream.min.js">Download stream.js</a>
+            <a class='download' href="stream.js.zip">Download stream.js</a>
```

Comment:
@dionyziz what is your position on this? Should we submit stream.js to [jsDelivr](https://www.jsdelivr.com/) and bundle the minified version with its source map or do something else.

But a remaining question if we don't bundle the two files, how would a user of the minified version debug stream.js without source map? Should we change the website to have two downloads?